command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
I think that we should require a runtime to implement the CLI otherwise there is no way to test compliance (like @wking said). cc: @RobDolinMS